merlin/systems/dag/runtimes/triton/ops/feast.py

```diff
@@ -0,0 +1,345 @@
+import json
+
+import numpy as np
+from feast import FeatureStore, ValueType
+
+from merlin.core.protocols import Transformable
+from merlin.dag import ColumnSelector
+from merlin.schema import ColumnSchema, Schema
+from merlin.systems.dag.ops.feast import QueryFeast
+from merlin.systems.dag.runtimes.triton.ops.operator import TritonOperator
+
+# Feast_key: (numpy dtype, is_list, is_ragged)
+feast_2_numpy = {
+    ValueType.INT64: (np.int64, False, False),
+    ValueType.INT32: (np.int32, False, False),
+    ValueType.FLOAT: (np.float32, False, False),
+    ValueType.INT64_LIST: (np.int64, True, True),
+    ValueType.INT32_LIST: (np.int32, True, True),
+    ValueType.FLOAT_LIST: (np.float32, True, True),
+}
+
+
+class QueryFeastTriton(TritonOperator):
+    """
+    The QueryFeast operator is responsible for ensuring that your feast feature store [1]
+    can communicate correctly with tritonserver for the ensemble feast feature look ups.
+
+    References
+    ----------
+    [1] https://docs.feast.dev/
+    """
+
+    @classmethod
+    def from_feature_view(
```

Comment:
It seems like a lot of this is the same as the definition of the method in the base class, so you might be able to call `super()` and then wrap the result with `QueryFeastTriton`

Reply:
The super class here is the TritonOperator, I am not sure that has from_feature_view? I would love to have the base operator as a parent class. Can I put more than one operator to inherit from? What would supersede what?